tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -207,9 +207,9 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
             if should_exclude_files_type(file_filepath, exclude_file_types):
                 LOGGER.info(
                     "Excluding {file_filepath} from glacier backup "
-                    "because of collection configured {CONFIG_EXCLUDE_FILE_TYPES_KEY}.",
+                    "because of collection configured {CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS}.",
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention

Reply:
updated